src/__/collections/some.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace collections;
+
+/**
+ * Equivalent to JavaScript's `Array.prototype.some()` function, this method will
+ * return true if any one item in the collection passes the callback truth test.
+ *
+ * This method will short-circuit on the first truthy callback result.
+ *
+ * **Usage**
+ *
+ * ```php
+ * __::some([1, 3, 5, 10, 7, 9], static function ($item, $key, $collection) {
+ *   return $item % 2 === 0;
+ * });
+ * ```
+ *
+ * **Result**
+ *
+ * ```
+ * true
+ * ```
+ *
+ * @since 0.2.4 added to library
+ *
+ * @template V
+ * @template K
+ *
+ * @param iterable|\stdClass $collection
+ * @param \Closure           $callback
+ *
+ * @return bool
+ */
+function some($collection, \Closure $callback)
```

Comment:
We may add `__::identity` as the default $callback, so we can just do `__:some` on a collection of booleans (or "trueables") without having to write/pass the callback.

This is the case in Lodash [`some`](https://lodash.com/docs/4.17.15#some), Elixir [`any?`](https://hexdocs.pm/elixir/1.12/Enum.html#any?/1) (but not in Clojure [`some`](https://clojuredocs.org/clojure.core/some), but there users are more familiar with functional composition).

If this is accepted `__:every` must also be updated.

Reply:
I can 100% get behind defaulting the callbacks to `__::identify()` like other libraries. Pushed up those changes 👍